docs/Report.md

```diff
@@ -17,59 +17,42 @@ Provide an illustration of your domain model. Make sure that it is correct and c
-</figure>
+![ULM Class Diagram](Images/PackageCoreUMLDiagram.png){width=60%}
+
+![ULM Class Infrastructure](Images/PackageInfrastructureUMLDiagram.png){width=60%}
+
+![ULM Class Razor](Images/PackageRazorUMLDiagrams.png){width=60%}
 
-<figure align = "center">
-    <img title="ULM Class Razor" style="width:30%" alt="Alt text" src="Images/PackageRazorUMLDiagrams.png">
-    <figcaption style="  font-size:11px"><b>Fig. XX - ULM Package Diagram of the Chirp.Razor </b></figcaption>
-</figure>
 
-<figure align = "center">
-    <img title="ULM Class Pages" style="width:30%" alt="Alt text" src="Images/PackagePagesUMLDiagram.png">
-    <figcaption style="  font-size:11px"><b>Fig. XX - ULM Package Diagram of the Chirp.Razor/Pages </b></figcaption>
-</figure>
+![ULM Class Pages](Images/PackagePagesUMLDiagram.png){width=60%}
 
 The Onion Architecture (otherwise known as Clean Architecture), is great for having low coupling and high cohesion. When looking at the UML in the more specified onion diagram bellow, there is no unnecessary communication between scripts, having low coupling making the readability of the program better, even though some of the repositories contain a fair amount of methods. When moving outward you'll see the packages only use entities further in or in the same layer.
 
 It is worth mentioning that the only way of interacting with the repositories is through their interfaces, which is an important factor in making sure the application has low coupling. The same goes for the CheepService, since every class that needs to access it uses information from the interface, and that interface uses from the other interfaces. 
 <br>
-<figure id="OnionClassDiagram">
-    <center><img src="Images\OnionClassDiagram.png" style="width:30%" alt="onion diagram">
-  <figcaption >Fig.XX OnionClassDiagram</figcaption></center>
-</figure>
+
+![OnionClassDiagram](Images/OnionClassDiagram.png){width=60%}
 
 ## Architecture of deployed application
 In the following figure a deployment diagram can be seen of our Chirp application.
 
-<figure>
-    <center><img src="Images/DeploymentDiagram.png" alt="Deploy diagram">
-  <figcaption>Fig.XX Deployment diagram</figcaption></center>
-</figure>
+
+![Deployment diagram](Images/DeploymentDiagram.png){width=60%}
 
 
-Chirp is a client-server application hosted on the Azure app service as a Web App. The web app is connected to a Azure SQL server where the database can be found. Furthermore the application makes use of a Azure AD B2C tenant for user-authentication. The different nodes means of communication is represented in the diagram.
+Chirp is a client-server application hosted on the Azure app service as a Web App. The web app is connected to an Azure SQL server where the database can be found. Furthermore the application makes use of an Azure AD B2C tenant for user-authentication. Each node and its means of communication are represented in the diagram. 
```

Comment:
Grammar and Restructured the last sentence to increase readability.

Reply:
looks good